static/src/server.ts

```diff
@@ -503,12 +505,13 @@ app.get("/nodejs/query", (req: Request, res: Response) => {
   const client = req.query.client || BARD_CLIENT_URL_PARAM;
   const allowedTileTypes =
     client === BARD_CLIENT_URL_PARAM ? BARD_ALLOWED_CHARTS : null;
+  const mode = req.query.mode || DEFAULT_NL_MODE;
   res.setHeader("Content-Type", "application/json");
   axios
     // Set "mode=strict" to use heuristic detector, disable using default place,
     // use a higher SV threshold and avoid multi-verb queries
     .post(
-      `${CONFIG.apiRoot}/api/explore/detect-and-fulfill?q=${query}&mode=strict&client=${client}`,
+      `${CONFIG.apiRoot}/api/explore/detect-and-fulfill?q=${query}&mode=${mode}&client=${client}`,
```

Comment:
Maybe follow up, could we maybe add SV/place DCIDs in the charts?  Without that now its hard to know which chart has what SVs.

Reply:
do you mean in the json response for each chart include additional fields for SV and place DCIDs?

Reply:
Yes.